### ISSUE TITLE
typescript-resolvers: Interface field with args doesn't generate argument type

### ISSUE THREAD

**Issue by derimagia** [dotansimha/graphql-code-generator#1563]
**Describe the bug**
An interface with a field with arguments doesn't generate the type for the arguments at all and thus errors because it is being referenced. Tested in 1.0.4 thinking it was fixed there but doesn't seem to be. 

May be an issue out of typescript-resolvers.

**To Reproduce**

From this repo in master in file `ts-resolvers.spec.ts`:
```diff
diff --git a/packages/plugins/typescript-resolvers/tests/ts-resolvers.spec.ts b/packages/plugins/typescript-resolvers/tests/ts-resolvers.spec.ts
index ba548c34..d6db3049 100644
--- a/packages/plugins/typescript-resolvers/tests/ts-resolvers.spec.ts
+++ b/packages/plugins/typescript-resolvers/tests/ts-resolvers.spec.ts
@@ -6,7 +6,11 @@ import { validateTs } from '../../typescript/tests/validate';
 
 describe('TypeScript Resolvers Plugin', () => {
   const schema = buildSchema(/* GraphQL */ `
-    type MyType {
+    interface MyTypeInterface {
+      withArgs(arg: String, arg2: String!): String
+    }
+    
+    type MyType implements MyTypeInterface {
       foo: String!
       otherType: MyOtherType
       withArgs(arg: String, arg2: String!): String

```

This fails the tests w/ error "Cannot find name `MyTypeInterfaceWithArgsArgs`."

**Expected behavior**
Generate valid typescript

**Environment:**

- OS: 
- `@graphql-codegen/1.0.4`: 
- NodeJS: 11.12.0

Really liking the new version by the way! Only have had a few issues upgrading

**Comment by kamilkisiela** [comment]
#1566 should fix it

**Comment by dotansimha** [comment]
Fixed in 1.0.5.
